pkg/meta/interface.go

```diff
@@ -142,4 +142,8 @@ type Meta interface {
 	Setlk(ctx Context, inode Ino, owner uint64, block bool, ltype uint32, start, end uint64, pid uint32) syscall.Errno
 
 	OnMsg(mtype uint32, cb MsgCallback)
+
+	// IsConfigSafe returns true if the configuration of the meta server
+	// met the persistence requirement for storing metadata.
+	IsConfigSafe() bool
```

Comment:
If we find that Redis is misconfigured, we should show an warning or even exit with -1 in inmediaetly, for example, 

```
 if !r. aofEnabled {
   logger.Warnf("AOF is not enabled, you may lose data if Redis is not shutdown properly.")
}
```

All of these checking could be done in pkg/redis/, so we don't need this API here.